### PR TITLE
Action: Datasheets render

### PR DESCRIPTION
**Pull request by marqdevx**
## What This PR Changes
- New GitHub Action to render the datasheets, and check if there are issues on the rendering

I did sometesting at https://github.com/marqdevx/docs-content-private/actions/runs/3524139452
(private repo)

### CODE REVIEW

File: scripts/datasheet-rendering/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "datasheet-rendering",
-  "version": "1.0.0",
```
Comment (sebromero) [inline scripts/datasheet-rendering/package.json:3]: This doesn't refer to the datasheet-renderer package and hence has no effect here.

**Comment by sebromero** [comment]
@marqdevx Looks good, what's the purpose though?

File: scripts/datasheet-rendering/package.json (hunk 1 above)
Comment (marqdevx) [reply scripts/datasheet-rendering/package.json:3]: True
What is that referring to?

Comment (sebromero) [reply scripts/datasheet-rendering/package.json:3]: The script that integrates the renderer is also a node module. One that's not published in the registry but still it's a module.

File: scripts/datasheet-rendering/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "1.0.2",
```
Comment (sebromero) [inline scripts/datasheet-rendering/package.json:3]:
```suggestion
  "version": "1.0.0",
```

**Review by sebromero — APPROVED** [review]
LGTM

**Pull request merged**